### PR TITLE
Add Investment Project comments.

### PR DESCRIPTION
**Pull request by elcct**
This adds `comments` field to Investment Project and migration command to migrate missing comments.

### CODE REVIEW

File: datahub/investment/serializers.py [hunk 1]
```diff
@@ -23,6 +23,7 @@ class IProjectSummarySerializer(serializers.ModelSerializer):
     """Serialiser for investment project endpoints."""
 
     incomplete_fields = serializers.SerializerMethodField()
+    comments = serializers.CharField(read_only=True)
```
Comment (reupen) [inline datahub/investment/serializers.py]: How about using read_only_fields on Meta? Most of the other read-only fields are specified as read-only there.

File: datahub/investment/test/test_views.py [hunk 2]
```diff
@@ -68,6 +68,7 @@ def test_list_projects_success(self):
             'project_code',
             'investment_type',
             'description',
+            'comments',
```
Comment (reupen) [inline datahub/investment/test/test_views.py]: Could test updating of comments in test_update_read_only_fields() as well.

File: datahub/investment/serializers.py (hunk 1 above)
Comment (reupen) [reply datahub/investment/serializers.py]: Also, maybe it should be read-only in admin as well.

File: datahub/dbmaintenance/test/commands/test_update_investment_project_comments.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        # should be ignored
+        InvestmentProjectFactory(),
+    ]
+
+    old_comments = investment_projects[1].comments
+
+    bucket = 'test_bucket'
+    object_key = 'test_key'
+    csv_content = f"""Id,comments
+00000000-0000-0000-0000-000000000000,Comment 1
```
Comment (reupen) [inline datahub/dbmaintenance/test/commands/test_update_investment_project_comments.py]: Might be nice to put a muiti-line comment in there (quoted).

File: datahub/investment/serializers.py (hunk 1 above)
Comment (elcct) [reply datahub/investment/serializers.py]: Good points. Done.

File: datahub/investment/test/test_views.py (hunk 2 above)
Comment (elcct) [reply datahub/investment/test/test_views.py]: Good idea. Done.

File: datahub/dbmaintenance/test/commands/test_update_investment_project_comments.py (hunk 3 above)
Comment (elcct) [reply datahub/dbmaintenance/test/commands/test_update_investment_project_comments.py]: Also good idea. Done.

**Comment by elcct** [comment]
Found a bug in `test_simulate` for another migration command and couldn't resist to fix it.

**Pull request merged**